content/java/concepts/strings/terms/equalsIgnoreCase/equalsIgnoreCase.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: '.equalsIgnoreCase()'
+Description: 'Compares strings in a case-insensitive manner.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'String'
+CatalogContent:
+  - 'learn-java'
+---
+
+The **`.equalsIgnoreCase()`** method compares strings and then then gives an output of true or false. It is not a case sensitive method.
+
+## Syntax
+
+```pseudo
+String nameOfYourString = "the value you assign";

```

Comment:
@SancHard. I think you've been misunderstanding my comment about the syntax block. I want you to simplify it so it is just one line illustrating the syntax. So the whole ```pseudo block reads something like this:

`boolean result = stringOne.equalsIgnoreCase(stringTwo);`

Then I just want a following paragraph explaining the two string variables and the result.

Let me know if you need assistance or further explanation.

Reply:
@SSwiniarski  I would like to apologize because I should have asked you in the first place, so I accept that it was my fault. I believe that I've finally understood what you wrote so made another update to the file. You'll see the updated part down below so you don't have to check the whole file again. Please let me know if I understood you better this time.

## Syntax

```pseudo
boolean result = stringOne.equalsIgnoreCase(stringTwo);
```
On the syntax above, we have three different variables. One of them being a boolean type variable, the other two being a string type variable. As mentioned on the description of this method, `.equalsIgnoreCase()` compares strings in a case insensitive way, so if the values stored in the strings are same, the result stored in `result` will be true. Otherwise, it will be false.